product_supplierinfo_for_customer/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
```

Comment:
Can you generate the readme using https://github.com/OCA/maintainer-tools/blob/master/tools/gen_addon_readme.py

Reply:
Ok, then we'll wait for the cron when it's merged.